public/js/office.web.js

```diff
@@ -264,7 +275,11 @@ $(() => {
     initGetUserMenu(officeEvents);
 
     officeEvents.onParticipantStartedMeet(showUserInRoom);
-    officeEvents.onParticipantLeftMeet(showUserInRoom);
+    officeEvents.onParticipantLeftMeet(function(user,roomId){
```

Comment:
To keep the same approach that we have in the `officeEvents.onParticipantStartedMeet` function, maybe we could create another function called

- `showUserLeaveRoom`

```javascript
function showUserLeaveRoom(user, roomID) {
  showUserInRoom(user,roomId);
  notifyRoomExited(user,roomId);
}
```

and use like this:

```javascript
officeEvents.onParticipantStartedMeet(showUserInRoom);
officeEvents.onParticipantLeftMeet(showUserLeaveRoom);
```

What do you think?